contacts/migrations/0002_test_data.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations, transaction
+from users.models import User, Apartment, Seeker
+from contacts.models import Connection
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('contacts', '0001_initial'),
+        ('users', '0006_test_data'),
+    ]
+
+    def generate_connection_data(apps, schema_editor):
+        connection_test_data = [
+            ('user3@hmail.com', 'user2@hmail.com'),
+            ('user3@hmail.com', 'user1@hmail.com'),
+        ]
+        with transaction.atomic():
+            for seeker_email, apartment_email in connection_test_data:
+                user_seeker = User.objects.get(email=seeker_email)
+                user_apartment = User.objects.get(email=apartment_email)
+                sample_seeker = Seeker.objects.get(base_user=user_seeker)
+                sample_apartment = Apartment.objects.get(owner=user_apartment)
+                Connection(apartment=sample_apartment, seeker=sample_seeker).save()
```

Comment:
I believe you can use:
`sample_seeker = Seeker.objects.get(user__email=seeker_email)`
To span the relationships. This will enable you to use only 2 statements instead of 4.
See https://docs.djangoproject.com/en/3.2/topics/db/queries/#lookups-that-span-relationships

Reply:
@simzacks - Gave it a shot and it worked great :) Thanks!
Please review again